static/css/_carousel.scss

```diff
@@ -0,0 +1,180 @@
+.carousel-title {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+
+    &__content {
+        max-width: 992px;
+    }
+
+    h1 {
+        text-align: center;
+        margin-top: 80px;
+        margin-bottom: 40px;
+        font: 700 3rem $font-IBM;
+        line-height: 4rem;
+        color: $header-color;
+
+        @include for-size(mobile) {
+            font: 700 1.5rem $font-IBM;
+            text-align: center;
+            padding: 0 2rem;
+        }
+    }
+}
+
+.carousel_section .slick-list {
+    margin: 0 0 0 120px;
+
+    @media (max-width: 1024px) {
+        margin: 0 0 0 2em; 
+    }
+  }
+
+.carousel_section {
```

Comment:
the name should be `carousel-section ` 
`_` underline is being used to mention the element and `--` for mentioning the modifier so to avoid confusion we usually dont use `_` for the root name 
http://getbem.com/naming/

Reply:
Thanks dear @sara-fs, we will create a new PR for these changes later 🙏